ci-operator/templates/cluster-launch-e2e-openshift-ansible.yaml

```diff
@@ -125,9 +125,18 @@ objects:
           -i test/ci/inventory \
           test/ci/launch.yml | tee -a /tmp/artifacts/ansible.log
 
-        ansible -vvv \
-          -i test/ci/inventory masters \
-          -m fetch -a 'src=/etc/origin/master/admin.kubeconfig dest=/tmp/shared/ flat=true'
+        # Create a new cluster-admin user
```

Comment:
You didn't restore the test container, so this PR isn't "restoring testing".

Reply:
Yes